.github/workflows/coverage.yml

```diff
@@ -28,14 +28,15 @@ jobs:
   # Job producing code coverage report
   # ==================================
   Code_coverage:
-    # When ubuntu version will be upgrade, you should look if xtensor is now available
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
 
     steps:
     - name: "Install libraries"
       run: |
         sudo apt-get update
-        sudo apt-get install lcov libboost-all-dev libhdf5-dev libeigen3-dev libopencv-dev libsz2 ninja-build
+        sudo apt-get install ninja-build lcov
+        sudo apt-get install libhdf5-dev libsz2 libboost-all-dev libeigen3-dev libopencv-dev
+        sudo apt-get install libxtensor-dev
```

Comment:
On one line, it wouldn't have to re-read all the metadata?

Reply:
I was wondering the same, but if I look here at the log, I think this is the start of the installing `libxtensor-dev`:
https://github.com/highfive-devs/highfive/actions/runs/13585850903/job/37980539915#step:2:1766
and this is the end:
https://github.com/highfive-devs/highfive/actions/runs/13585850903/job/37980539915#step:2:1829
If correct, it takes less than 4s for the whole operation. Reading metadata isn't slowing us down. Therefore, I'll take it over line continuation magic :)

Thank you for reviewing.